example/azure-pipelines.yml

```diff
@@ -0,0 +1,179 @@
+    endpoint: Bitbucket - sistemas
+    name: situm/sys-kubernetes-templates.git
+
+jobs: 
+- job:
+  displayName: Build ${{ parameters.system }}
+  pool:
+    vmImage: ${{ parameters.system }}
+  steps:
+  - checkout: self
+    fetchDepth: 1
+    fetchTags: false
+    displayName: Clone repository
+
+  - task: NodeTool@0
+    inputs:
+      versionSource: 'spec' # 'spec' | 'fromFile'. Required. Source of version. Default: spec.
+      versionSpec: '18.14.2'
+
+  - bash: |
+
+      echo -e "\n[+] Installing NPM version $(npmVersion)\n"
+      npm install -g $(npmVersion)
+
+
+      echo -e "\n[+] Install plugin dependencies\n"
+      npm install
+
+      echo -e "\n[+] Install example dependencies\n"
+      cd example
+      npm install
+
+    displayName: Install plugin dependencies
+
+  - bash: |
+
+      if [[ "${{ parameters.system }}" == *"macOS-"* ]];then
+          echo -e "\n[+] Install gnu-sed\n"
+          brew install gnu-sed
+          export PATH="/usr/local/opt/gnu-sed/libexec/gnubin:$PATH"
+      fi
+
+      echo -e "\n[+] Variables to set:"
+      echo -e "\t[+] X-API-EMAIL: ${{ parameters.XAPIEMAIL }}"
+      echo -e "\t[+] BUILDINGID: ${{ parameters.BUILDINGID }}"
+      echo -e "\t[+] FLOORID: ${{ parameters.FLOORID }}"
+
+      echo "export const SITUM_EMAIL = '${{ parameters.XAPIEMAIL }}';" > example/src/situm.tsx
+      echo "export const SITUM_API_KEY = '${{ parameters.XAPIKEY }}';" >> example/src/situm.tsx
+      echo "export const SITUM_BUILDING_ID = '${{ parameters.BUILDINGID }}';" >> example/src/situm.tsx
+      echo "export const SITUM_FLOOR_ID = '${{ parameters.FLOORID }}';" >> example/src/situm.tsx
+      
+      sed -i "s/HERE_GOES_GOOGLE_MAPS_API_KEY/$(MAPS_APIKEY_EXAMPLE)/" example/ios/example/AppDelegate.mm
+      sed -i "s/HERE_GOES_GOOGLE_MAPS_API_KEY/$(MAPS_APIKEY_EXAMPLE)/" example/android/app/src/main/AndroidManifest.xml
+
+    displayName: Configure credentials
+  
+  - ${{ if contains(parameters.system, 'macOS') }}:
+
+    - task: DownloadSecureFile@1
+      name: api_key
+      inputs:
+        secureFile: api_key.json
+
+    - task: InstallAppleCertificate@2
+      inputs:
+        certSecureFile: AppleCert23.p12
+        
+    - task: InstallAppleCertificate@2
+      inputs:
+        certSecureFile: DistributorWildcard.p12
+  
+    - task: InstallAppleProvisioningProfile@1
+      inputs:
+        provisioningProfileLocation: 'secureFiles'
+        provProfileSecureFile: RNSDKExample.mobileprovision
+        removeProfile: true
+    
+    - task: InstallAppleProvisioningProfile@1
+      inputs:
+        provisioningProfileLocation: 'secureFiles'
+        provProfileSecureFile: DistributorOnline.mobileprovision
+        removeProfile: true
+  
+    - bash: |
+        echo -e "\n[+] Install iOS example dependencies\n"
+        cd example/ios
+        pod install
+      displayName: Install iOS example dependencies
+
+    - bash: |
+
+        sudo xcode-select -switch /Applications/Xcode_14.2.app
```

Comment:
Cada ano hai que cambiar a version de macos e xcode e vaino haber que facer para todolos proxectos. Imaxino que non se podera unificar para todolos proxectos, pero polo menos se poden quedan as duas variables seguidas no pipelines quedaría máis comodo

Reply:
Incluida como variable da pipeline